content/docs/introduction/requirements.md

```diff
@@ -146,6 +146,7 @@ The table below shows which version of mSupply you will require when running ope
 | :--------------- | :-------------------- |
 | 1.1.00 - 1.1.16  | 7.04.01+              |
```

Comment:
Should this be 1.1.00+ or should we change the later entries to describe a range? or leave it as it is?

Reply:
leave as it: anything later than 1.1.16 requires mSupply to be 7.05.05 or above